src/stories/Library/Modals/modal-loan/ModalLoan.tsx

```diff
@@ -244,7 +186,7 @@ export const ModalLoan: React.FC<ModalLoanProps> = ({
             />
             <Button
               buttonType="default"
-              label={`Forny mulige (${countRenewable})`}
+              label="Forny mulige (2)"
```

Comment:
Is (2) supposed to be hardcoded?

Reply:
I removed countRenewable because this is a design system, and I think it is unnecessary complexity to maintain.